### PR TITLE
admins can now trigger any random event via the secrets tab

### PR DESCRIPTION
**Pull request by MadmanMartian**
needed something to help test out the random event centcomm order in #16757 , and thought it might be useful for others.

Not sure if more security checks may be required, such as flag checks, so input is requested.

### CODE REVIEW

**Comment by Aurx** [comment]
It was a "random" event, yes, ask fewer questions goy.  

Does this log both who hit it, and what exact event they triggered?

**Comment by N3X15** [comment]
Nope, it doesn't.

Add logs plx.

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -3319,6 +3319,12 @@
 					var/mobtype = input("What mob would you like?", "Mob Swarm") as null|anything in typesof(/mob/living)
 					message_admins("[key_name_admin(usr)] triggered a mob swarm.")
 					new /datum/event/mob_swarm(mobtype, amt)
+			if("pick_event")
+				feedback_inc("admin_secrets_fun_used",1)
+				feedback_add_details("admin_secrets_fun_used","ALL")
+				var/choice = input("Which event do you want to trigger?") in subtypesof(/datum/event)+"Cancel"
+				if(choice != "Cancel")
+					new choice
```
Comment (N3X15) [inline code/modules/admin/topic.dm:3327]: This code needs a `message_admins()` call so admins know who did what.

**Review by N3X15 — DISMISSED** [review]

**Comment by MadmanMartian** [comment]
@N3X15 message added

**Pull request merged**